.env.example

```
@@ -1,4 +1,8 @@
 VITE_API_TOKEN=before josue
 VITE_API_V2_TOKEN=after josue
 VITE_BASE_URL=
-VITE_BASE_URL_API_V2=
\ No newline at end of file
+VITE_BASE_URL_API_V2=
+VITE_DEPLOYE_API_URL=
+#VITE_DEPLOYE_API_URL, su valor es un puerto expuesto para que las imagenes enviadas al correo y vistas en el pdf no generen error, ya que si las pongo localhost, el navegador introduce una url por seguridad asi dañanado la url donde se muestra la imagen
+VITE_PORTADA_API_URL=
+#VITE_PORTADA_API_URL= es la url de conexion con el archivo que se encarga de generar el pdf (generator-pdf.php)
```

```suggestion
VITE_PORTADA_API_URL=la url de conexion con el archivo que se encarga de generar el pdf (generator-pdf.php)
```

Se que no está normalizada la documentación. pero me apetece decir que esa forma de escribir es redundante e incómoda.